pkg/arvo/sys/hoon.hoon

```diff
@@ -12083,6 +12083,21 @@
   ^-  vase
   [[%cell p.hed p.tal] [q.hed q.tal]]
 ::
+++  side                                                ::  split vase to cell
+  |=  vax=vase
+  ^-  [hed=vase tal=vase]
+  ?@  q.vax  !!
+  =.  p.vax  (~(fuse ut p.vax) [%cell %noun %noun])
+  [(slot 2 vax) (slot 3 vax)]
+::
+++  sill                                                ::  split vase to list
+  |=  vax=vase
+  ^-  (list vase)
+  ?~  q.vax
```

Comment:
This is really just `?:  =(0 q.vax)`. I wonder if we should be checking that the type is `~` as well:

```
?>  (~(nest ut [%atom %n `0]) & p.vax)
```

But I'm not sure how much it matters...